tap_hubspot/__init__.py

```diff
@@ -945,7 +952,7 @@ class Stream(object):
     # Do these last as they are full table
     Stream('forms', sync_forms, ['guid'], 'updatedAt', 'FULL_TABLE'),
     Stream('workflows', sync_workflows, ['id'], 'updatedAt', 'FULL_TABLE'),
-    Stream('owners', sync_owners, ["ownerId"], 'updatedAt', 'FULL_TABLE'),
+    Stream('owners', sync_owners, ["id"], 'updatedAt', 'FULL_TABLE'),
```

Comment:
I wonder how Stitch is going to handle this change 🤔  We might need to delete the entire `owners` table and re-sync it completely

Reply:
I think that will be the case